sample_apps/simple_app_under_sev/sev_example_app.mak

```diff
@@ -29,11 +29,13 @@ S= $(SRC_DIR)/src
 O= $(OBJ_DIR)
 US=.
 I= $(SRC_DIR)/include
-INCLUDE=-I $(I) -I/usr/local/opt/openssl@1.1/include/ -I $(S)/sev-snp
+INCLUDE=-I $(I) -I/usr/local/opt/openssl@1.1/include/ -I $(S)/sev-snp -I.
+COMMON_SRC = ../common
```

Comment:
Could you move to specifying dirs off of CERTIFIER_ROOT; i.e. this becomes

COMMON_SRC = $(CERTIFIER_ROOT)/sample_apps/common